Configurations/unix-Makefile.tmpl

```diff
@@ -960,7 +965,7 @@ reconfigure reconf:
           my $ord_ver = $args{intent} eq 'lib' ? ' --version $(VERSION)' : '';
           my $ord_name = $args{generator}->[1] || $args{product};
           return <<"EOF";
-$target: @{[quotify_dep($args{generator}->[0])]} $deps \$(subst \$(subst x, ,x),\\ ,\$(SRCDIR)/util/mkdef.pl)
+$target: @{[quotify_dep($args{generator}->[0])]} $deps \$(SRCDIRINRULE)/util/mkdef.pl)
```

Comment:
I think you forgot to remove `)` at the end.

Reply:
Indeed, nice catch! Fixed and rebased to master to avoid a new conflict.